app/views/documents/show/_history_tab.html.erb

```diff
@@ -11,6 +11,14 @@
       <%= t "documents.history.entry_types.#{entry.entry_type}" %>
     </h4>
 
+    <div class="app-timeline-entry__dateline">
+      <%= entry.created_at.strftime("%d %B %Y at %l:%M%P") %> by <%= entry.username_or_unknown %>
+    </div>
+
+    <% if entry.internal_note.present? %>
+      <p><%= entry.internal_note.body %></p>
```

Comment:
Sorry this is probably only mentioned on the [epic](https://trello.com/c/d2ukJ9kE/592-internal-notes_) rather than the story - my bad

but this should support preserving line breaks and convert urls automatically into links

Reply:
👍